BasicRdl/Views/CategoryBrowser.xaml.cs

```diff
@@ -47,8 +47,13 @@ public CategoryBrowser(bool initializeComponent)
             if (initializeComponent)
             {
                 this.InitializeComponent();
-                FilterStringService.FilterString.AddGridControl(this.CategoriesGridControl);
+                this.Control = this.CategoriesGridControl;
```

Comment:
I think the name "Control" is too generic.
When you look at the IPanelFilterableDataGridView interface, it's a logical thing, but in the implementations I think 
FilterableControl (or something like that) says more.